### ISSUE TITLE
When web browser launches, it should launch with Ip address rather than "Localhost"

### ISSUE THREAD

**Issue by Techno11** [FIRST-Tech-Challenge/scorekeeper#109]
I can see non-tech-savvy people typing in "localhost" to an audience display (or any other computer) only having it go to no where.

**Comment by Nate2019** [comment]
I agree with that! The first type I set it up, I did type in "localhost" on the iPads for the ref tablets, and then quickly realized I was an idiot and needed the IP address....
